scripts/dependencies/tools.ts

```diff
@@ -147,10 +147,15 @@ export class KuberlrAndKubectl implements Dependency {
     await fs.promises.symlink('kuberlr', binKubectlPath);
   }
 
-  async getAvailableVersions(): Promise<string[]> {
+  async getAvailableVersions(includePrerelease = false): Promise<string[]> {
     const tagNames = await getPublishedReleaseTagNames(this.githubOwner, this.githubRepo);
+    const allVersions = tagNames.map((tagName: string) => tagName.replace(/^v/, ''));
+
+    if (!includePrerelease) {
+      return allVersions.filter(version => semver.prerelease(version) === null);
+    }
```

Comment:
This bit is being repeated over and over; would it make sense to extract it to a different helper function (like `getPublishedReleaseTagNames` is)?  Best thing I could think of is `getPublishedVersions(owner: string, repo: string, includePrerelease = false)`.

Reply:
I introduced `getPublishedVersions` as suggested.